dapps/templates/demo/config/blockchain.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   },
 
   privatenet: {
+    client: 'geth',
```

Comment:
Should the privatenet client be set to `"geth"` in the other template/test dapps' blockchain configs as you've done in this one?

Reply:
Updated. I backtracked on my first config change. Instead of putting Ganache as the definitive default, I let Geth there, but put Ganache from development, because after thinking about it, using Ganache for anything other then development and tests is kind of weird/not possible.

Anyway, let me know if it  makes sense. Anyway, `development` is the default of all commands, so it still serves the use case of new users start Embark not needing Geth

Reply:
With the new arrangement we don't need L18, is that correct?

Reply:
Correct. Deleted